lms/djangoapps/discussion_api/serializers.py

```diff
@@ -290,6 +290,7 @@ class CommentSerializer(_ContentSerializer):
     endorsed_by = serializers.SerializerMethodField()
     endorsed_by_label = serializers.SerializerMethodField()
     endorsed_at = serializers.SerializerMethodField()
+    child_count = serializers.IntegerField(read_only=True)
```

Comment:
@nasthagiri we may or may not get children from forums as we can always set `recursive=False` to improve performance whenever we do not require child comments.